providers/bundles/org.eclipse.ecf.provider.filetransfer.httpclientjava/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,67 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %plugin.name
+Bundle-SymbolicName: org.eclipse.ecf.provider.filetransfer.httpclientjava;singleton:=true
+Bundle-Version: 1.0.300.qualifier
+Bundle-Vendor: %plugin.provider
+Bundle-Localization: plugin
+Automatic-Module-Name: org.eclipse.ecf.provider.filetransfer.httpclientjava
+Bundle-RequiredExecutionEnvironment: JavaSE-11
+Require-Bundle: org.eclipse.equinox.common;bundle-version="3.13.0",
+ org.eclipse.ecf;bundle-version="3.9.101",
+ org.eclipse.core.jobs;bundle-version="3.10.800"
+Import-Package: javax.net.ssl,
+ org.apache.hc.client5.http;version="5.0.2",
```

Comment:
This looks like copy paste error, right? Apache httpclient shouldn't be needed by this bundle.

Reply:
Currently this is just a copy of the hc5 bundle with just package/project names adjusted.

Reply:
Ah, I thought it's implementation using jvm's httpclient.

Reply:
Yes that's the goal, but currently it has no changed implementation and I'm using this as a base to investigate what needs changed.